src/mimefactory.rs

```diff
@@ -18,29 +18,30 @@ use crate::param::*;
 use crate::peerstate::{Peerstate, PeerstateVerifiedStatus};
 use crate::stock::StockMessage;
 
-#[derive(Clone, Copy, Eq, PartialEq)]
+#[derive(Clone)]
```

Comment:
Didn't we have something enforcing Debug impls on all structs/enums?

Reply:
It is not always possible, we have `DebugStub` derived in some places. In any case, I have added `Debug` to `Loaded` and `MimeFactory` now.